resource/tasks.json

```diff
@@ -3161,16 +3185,17 @@
         "next": ["Mall@ReturnButtons#next", "Mall", "Mall@CloseAnnos#next"]
     },
     "Mall": {
-        "algorithm": "OcrDetect",
+        "template": ["Mall.png", "MallDark.png", "MallSami.png", "MallMistCity.png", "MallSiege.png", "MallSarkaz.png"],
```

Comment:
```suggestion
        "template": ["MallOrigin.png", "MallDark.png", "MallSami.png", "MallMistCity.png", "MallSiege.png", "MallSarkaz.png"],
```

Reply:
Unless you want to change the template name, which is probably better as that would follow the format we've been using already.